core-plugin/src/com/google/gct/idea/debugger/CloudBreakpointHandler.java

```diff
@@ -49,22 +49,27 @@
 import javax.swing.SwingUtilities;
 
 /**
- * The Cloud Debugger breakpoint handler is responsible for creating or deleting the breakpoints on the server. It is
- * responsible for creating local IDE representations given a server representation. It stores a relationship between
- * the local IDE representation and the server breakpoint and provides methods for getting one given the other. {@link
- * XBreakpointHandler} is often called by intelliJ when the user enables or disables breakpoints.
- * <p/>
- * This class is threadsafe. All operations can be called by multiple threads.
- * <p/>
- * Note there are three breakpoint types managed in the debugger and are named to avoid confusion:
- * xIdeBreakpoint : these are {@link XBreakpoint} types that intelliJ defines and are sealed.
- * cloudIdeBreakpoint: these are our IDE defined customized breakpoints
- *                 {@link com.google.gct.idea.debugger.CloudLineBreakpointType.CloudLineBreakpoint} with
- *                 custom properties, etc. It's confusing that there are two hierarchies for IDE breakpoints,
- *                 but intelliJ does this to control some base behavior.
- * serverBreakpoint: these are the {@link Breakpoint} breakpoints our server gives us via the apiary api.
+ * The Cloud Debugger breakpoint handler is responsible for creating or deleting the breakpoints on
```

Comment:
can you call out if there are any actual text changes here apart from the reformat?